drivers/cc110x/cc110x-netdev.c

```diff
@@ -114,6 +114,9 @@ static int _get(netdev_t *dev, netopt_t opt, void *value, size_t value_len)
             return 1;
         case NETOPT_IPV6_IID:
             return _get_iid(dev, value, value_len);
+        case NETOPT_ADDR_LEN:
```

Comment:
Can you please also add `case NETOPT_SRC_LEN:` here? In case of `cc110x` it should return the same value and could prevent future problems with that option.

Reply:
like that:
`        case NETOPT_ADDR_LEN:
        case NETOPT_SRC_LEN:`
?

Reply:
We do it usually in two lines, but yes.

```C
case NETOPT_ADDR_LEN:
case NETOPT_SRC_LEN:
```